### PR TITLE
add access origin * tag

### PR DESCRIPTION
**Pull request by TommyJ1994**
- add access origin * tag to allow the mobile app to communicate with the cluster it was built on.
- regenerate shrinkwrap.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wfm2",
-  "version": "2.3.0",
+  "version": "2.4.0",
```
Comment (vchepeli) [inline package.json]: its not this minor fix?

File: config.xml [hunk 2]
```diff
@@ -8,6 +8,7 @@
       Red Hat Mobile
   </author>
   <content src="index.html"/>
+  <access origin="*"/>
   <access origin="https://fonts.googleapis.com"/>
```
Comment (vchepeli) [inline config.xml]: think you can remove all <access origin="https://* />

Comment (TommyJ1994) [inline config.xml]: Any reason why?

**Comment by jcesarmobile** [comment]
LGTM

File: config.xml (hunk 2 above)
Comment (vchepeli) [inline config.xml]: <access origin="*"/> allows all origins in this case you dont need to be specific and you can use only `*`
to be more specific you need use something like <access origin="https://*.feedhenry.com"/>

Comment (vchepeli) [inline config.xml]: And I dont see any difference having/removing other origins in mobile app

Comment (jcesarmobile) [inline config.xml]: @vchepeli studio will replace  the `*` in `<access origin="*"/>` with the cloud domain on app creation, that's why the other access tags are needed

**Pull request merged**